build.gradle

```diff
@@ -69,21 +69,21 @@ dependencies {
     compile group: 'org.scala-lang', name: 'scala-library', version: scalaVersion
     compile group: 'org.scala-lang', name: 'scala-reflect', version: scalaVersion
     compile group: 'org.scala-lang', name: 'scala-compiler', version: scalaVersion
-    compile group: 'org.scala-lang.modules', name: 'scala-java8-compat_2.11', version: scalaJava8CompatVersion
-    compile group: 'com.jsuereth', name: 'scala-arm_2.11', version: scalaArmVersion
+    compile group: 'org.scala-lang.modules', name: 'scala-java8-compat_2.12', version: scalaJava8CompatVersion
```

Comment:
Create a variable named sparkScalaVersion with value "2.12". Then replace use it like `'scala-java8-compat_'+sparkScalaVersion`.